### PR TITLE
Circle CI (macOS deployment)

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      - aws s3 sync dist/CellProfiler.dmg s3://cellprofiler-releases --delete
+general:
+  artifacts:
+    - dist/CellProfiler.app
+machine:
+  xcode:
+    version: 8.3
+test:
+  override:
+    - echo "foo"
```
Comment (0x00b1) [inline circle.yml]: This will be changed to `pytest`.

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    - /usr/local/lib/python2.7/site-packages
+  override:
+    - pip install --editable .[test] --upgrade
+  pre:
+    - brew update
+    - brew install mysql wxpython
+    - pip install --upgrade cython numpy scipy
+deployment:
+  nightly:
+    branch: features/circle
```
Comment (0x00b1) [inline circle.yml]: This will be changed to `master`.

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  pre:
+    - brew services start mysql
+dependencies:
+  cache_directories:
+    - /usr/local/lib/python2.7/site-packages
+  override:
+    - pip install --editable .[test] --upgrade
+  pre:
+    - brew update
+    - brew install mysql wxpython
```
Comment (0x00b1) [inline circle.yml]: Missing `upx`

**Comment by mcquin** [comment]
Moved to CellProfiler/distribution.

**Pull request closed without merging**